Makefile

```diff
@@ -91,48 +88,77 @@ version:
 .PHONY: docker_run
 docker_run:
 	docker run --rm -it \
-		-e CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE=${CREDENTIALS_PATH} \
+		-e PROJECT_ID \
+		-e REGION \
+		-e ZONE \
+		-e SERVICE_ACCOUNT_EMAIL \
+		-e SERVICE_ACCOUNT_JSON \
 		-e GOOGLE_APPLICATION_CREDENTIALS=${CREDENTIALS_PATH} \
 		-v $(CURDIR):/cft/workdir \
```

Comment:
Can we quote `$(CURDIR)` while we're at it, per https://github.com/terraform-google-modules/terraform-google-module-template/pull/21?